tests/chocolatey-tests/commands/choco-install.Tests.ps1

```diff
@@ -521,14 +534,26 @@ Describe "choco install" -Tag Chocolatey, InstallCommand {
             $XML.package.metadata.version | Should -Be "1.0.0"
         }
 
-        It "Should not have been able to delete the rollback" -Tag FossOnly, ExpectBroken {
+        It "Should not have been able to delete the rollback" -Tag Broken {

```

Comment:
I removed it as the behavior is different between licensed and FOSS, we should look at making the behavior the same for both.
To prevent that being ignored again, I removed the FossOnly flag.